edb/server/mng_port/edgecon.pyx

```diff
@@ -736,12 +754,19 @@ cdef class EdgeConnection:
         object io_format,
         bint expect_one,
         uint64_t implicit_limit,
-    ):
+    ) -> CompiledQuery:
         if self.debug:
             self.debug_print('PARSE', eql)
 
+        entry = rewrite(eql)
```

Comment:
I think "preparse" (tokenize+extract args) would be a better term for this stage.  Also, `Entry` is nondescript. `PreparsedQuery` seems better.

Reply:
Well we have [preparser module](https://github.com/edgedb/edgedb/blob/master/edb/edgeql-parser/src/preparser.rs) which is basically statement delimiter.

Reply:
How about "normalize" then?

Reply:
Renamed.